use-cases/predictive_maintenance/dw_flow/data/example_fleet_info.csv

```diff
@@ -0,0 +1,91 @@
+vehicle_id,make,model,year,vehicle_class,engine_type
```

Comment:
Did you mean to commit the 2 CSV files in this PR? If so, can they be hosted in S3 instead?

Reply:
Yes, I did - and yes, they can be hosted in S3 as well - I will work on that.